### ISSUE TITLE
Umami tracking script from cloned repository causing misdirected analytics

### ISSUE THREAD

**Issue by Paulj1989** [hcostax/hcostax_dot_com#1]
This repository appears to be cloned from my [website repository](https://github.com/paulj1989/paulrjohnson.net), and there are still quite a few artefacts from my site left over in your codebase, some of which are causing some issues on my end.

I'm assuming that your site is a work in progress and some of the more obvious similarities in wording and style will be cleaned up in the future, so generally speaking, it is not a problem, but there is one particular artefact that is confusing my website analytics. You have retained the `/assets/umami.html` script in your repository but changed the `data-domain` from `paulrjohnson.net` to `hcostax.com`. This is a tracking code and it seems that changing it to your website domain is causing some of your traffic to be picked up in my analytics.

I've been able to narrow it down to your homepage because the `index.qmd` in the root of your repository includes a line of code for adding this tracking to the homepage (`include-in-header: assets/umami.html`). I identified the problem because there was a disproportionate amount of traffic coming from Brazil, and from sources (Instagram, rogue-scholar.com) that I wouldn't expect to refer to my site often (if at all), and have been able to confirm this by following the link to your website in your Instagram profile to see if it gets picked up by my analytics (it does).

It looks like you're using Google Analytics, so I don't think you have any need for the `umami.html` script. If that's the case, deleting that file (and the `include-in-header: assets/umami.html` line, since this would be obsolete) should fix things. If you are using Umami tracking, you would need to set up your own tracking code, and change the `data-website-id` to your own ID (and keep both the file and line of code referencing the file).

**Comment by hcostax** [comment]
Hello Paul, how are you?
I apologize profusely. I didn't mean to cause you any problems.
And yes, I'm still editing.
I had some personal problems and stopped editing/testing in the middle of the process.
I did what you advised, and I believe that this problem should now stop. And I apologize for cloning without due credit. I'm still new to this github thing

**Comment by Paulj1989** [comment]
No worries @hcostax. It hasn't had a huge impact, I just spotted some traffic coming through that seemed unlikely, and figured out the issue when I investigated. 

Acknowledgement is always appreciated but no rush. I'm sure your energy can be better spent working on your site! I'm pleased you liked my website enough to want to use it as a starting point for your own, and it's always nice to see people benefiting from sharing code in the open. Feel free to give me a shout if you have any questions that I might be able to help with! 

I'll keep an eye out and if I spot any signs that the problem isn't resolved I'll let you know. I think this should have fixed the issue though.
